src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java

```diff
@@ -28,6 +28,7 @@
 
     @Value("${idam.url}") private String idamUrl;
     @Value("${s2s.url}") private String s2sUrl;
+    @Value("${maxAge.default}") private int maxAgeDefault;
```

Comment:
Regarding naming `maxAgeDefault` or `maxAge`, I'm wondering whether it would be sensible to add some information about the unit, i.e. `maxAgeInDays` or something. What do you think?

Reply:
Why not `storeForNoOfDays`?

Reply:
I'll see if Spring can accept something like `"2 days"` which then maps to Java's `Duration`

Reply:
just realised that `maxAge` is not the best name as it suggests it's time from when draft was created, not when it was last updated...  
Perhaps `maxStaleDays`? What do you thinks? Also, github needs polls ;)

Reply:
`maxStaleDays` sounds good to me.